src/Components/App/App.js

```diff
@@ -186,60 +191,81 @@ class App extends Component {
       }
       possibleSuggestions = this.gameFilter(activity, answers);
     }
-    return possibleSuggestions;
-  };
+    this.setState({possibleSuggestions: possibleSuggestions})
+    return possibleSuggestions
+  }
 
   render() {
     return (
       <div className="App">
         <Switch>
-          <Route exact path="/">
+          <Route exact path='/'>
             <Homepage
               getActivityData={this.getActivityData}
               allMovies={this.state.movies}
             />
           </Route>
-          <Route exact path="/form">
-            <Form
-              goBack={this.goBack}
-              setHistory={this.setHistory}
-              resetState={this.resetState}
-              activities={this.state.activities}
-              setActivities={this.setActivities}
-              updateActivityAnswers={this.updateActivityAnswers}
-              determineRandomActivity={this.determineRandomActivity}
-            />
-          </Route>
-          {/* <Route exact path='/result'>
-            <Result determineRandomActivity={this.determineRandomActivity}/>
-          </Route> */}
-          <Route
-            exact
-            path="/:activity"
-            render={({ match }) => {
+          <Route exact path="/form"
+            render={( {history} ) => {
               return (
-                <BrowsePage
-                  name={match.params.activity}
-                  data={this.state[match.params.activity]}
+                <Form
+                  route={history}
+                  goBack={this.goBack}
+                  setHistory={this.setHistory}
+                  resetState={this.resetState}
+                  activities={this.state.activities}
+                  setActivities={this.setActivities}
+                  updateActivityAnswers={this.updateActivityAnswers}
+                  determineRandomActivity={this.determineRandomActivity}
```

Comment:
Are we still using `determineRandomActivity()` as a prop in `Form`?

Reply:
If we are, I'm not sure that we necessarily need to, we have the redirect happening on submit and could just have this logic happen in app